src/Onboarding/OnboardingScreen.tsx

```diff
@@ -62,92 +57,174 @@ const OnboardingScreen: FunctionComponent<OnboardingScreenProps> = ({
-        </View>
-      </SafeAreaView>
+          </View>
+        </ScrollView>
+        <TouchableOpacity
+          style={style.bottomButtonContainer}
+          onPress={handleOnPressProtectPrivacy}
+        >
+          <GlobalText style={style.bottomButtonText}>
+            {t("onboarding.protect_privacy_button")}
+          </GlobalText>
+          <SvgXml
+            xml={Icons.ChevronUp}
+            fill={Colors.primary100}
+            width={Iconography.xxSmall}
+            height={Iconography.xxSmall}
+          />
+        </TouchableOpacity>
+      </View>
     </>
   )
 }
 
+const createStyle = (insets: EdgeInsets) => {
+  const iosPaddingTop = 65
+  const androidPaddingTop = 90
+  const headerHeight = Platform.select({
+    ios: iosPaddingTop,
+    android: androidPaddingTop,
+    default: iosPaddingTop,
+  })
+
+  /* eslint-disable react-native/no-unused-styles */
+  return StyleSheet.create({
+    header: {
+      position: "absolute",
+      width: "100%",
+      zIndex: Layout.zLevel1,
+      flexDirection: "row",
+      alignItems: "flex-end",
+      justifyContent: "space-between",
+      paddingTop: insets.top,
+      paddingHorizontal: Spacing.xSmall,
+      backgroundColor: Colors.primaryLightBackground,
+      borderBottomColor: Colors.neutral10,
+      borderBottomWidth: Outlines.hairline,
+    },
+    languageButtonContainer: {
+      borderRadius: Outlines.borderRadiusMax,
+      paddingVertical: Spacing.xxSmall,
+      paddingHorizontal: Spacing.xLarge,
+      marginBottom: Spacing.xSmall,
+    },
+    languageButtonText: {
+      ...Typography.body3,
+      letterSpacing: Typography.largeLetterSpacing,
+      color: Colors.primary125,
+      textAlign: "center",
+      textTransform: "uppercase",
+    },
+    skipButtonText: {
+      ...Typography.body2,
+      color: Colors.neutral100,
+      padding: Spacing.medium,
+    },
+    outerContainer: {
+      justifyContent: "space-between",
+      flex: 1,
+      backgroundColor: Colors.primaryLightBackground,
+    },
+    contentContainer: {
+      flexGrow: 1,
+      justifyContent: "space-between",
+      paddingTop: headerHeight + Spacing.medium,
+      paddingBottom: Spacing.xxLarge,
+    },
+    image: {
+      width: "97%",
+      height: 220,
+      marginBottom: Spacing.medium,
+    },
+    headerText: {
+      ...Typography.header1,
+      marginBottom: Spacing.xLarge,
+      paddingHorizontal: Spacing.large,
+    },
+    nextButtonContainer: {
+      alignSelf: "flex-start",
+      paddingHorizontal: Spacing.large,
+    },
+    bottomButtonContainer: {
+      backgroundColor: Colors.secondary10,
+      flexDirection: "row",
+      alignItems: "center",
+      justifyContent: "center",
+      paddingTop: Spacing.small,
+      paddingBottom: insets.bottom + Spacing.small,
+    },
+    bottomButtonText: {
+      ...Typography.header5,
+      color: Colors.primary100,
+      marginRight: Spacing.xSmall,
+    },
+  })
+}
+
+export default OnboardingScreen
```

Comment:
Nbd on this one, but its convention to have the exports at the bottom of the file.